src/point.js

```diff
@@ -149,6 +149,18 @@ $.Point.prototype = {
     },
 
     /**
+     * Compute the squared distance between this point and another point.
+     * Useful for optimizing things like comparing distances.
+     * @function
+     * @param {OpenSeadragon.Point} point The point to compute the distance with.
+     * @returns {Number} The distance between the 2 points
```

Comment:
Sorry, I think your doc comment is already great. I was just referring to this line: 

```
* @returns {Number} The distance between the 2 points
```

... which should be:

```
* @returns {Number} The squared distance between the 2 points
```

...for accuracy.